src/index.ts

```diff
@@ -481,17 +482,21 @@ class Spanner extends GrpcService {
           displayName,
           nodeCount: config.nodes,
           processingUnits: config.processingUnits,
+          autoscalingConfig: config.autoscalingConfig,
```

Comment:
We need to do changes for [Update Instance](https://github.com/googleapis/nodejs-spanner/blob/main/src/instance.ts#L1598) to handle below use case 
When autoscaling config is set then node and processing unit should not be passed